applications/launchpad/backend/src/docker/mod.rs

```diff
@@ -57,3 +62,90 @@ lazy_static! {
 lazy_static! {
     pub static ref CONTAINERS: RwLock<HashMap<String, ContainerState>> = RwLock::new(HashMap::new());
 }
+
+pub fn tari_blockchain_volume_name(tari_workspace: &str, tari_network: TariNetwork) -> String {
+    format!("{}_{}_volume", tari_workspace, tari_network.lower_case())
+}
+
+pub fn network_name(tari_workspace: &str) -> String {
+    format!("{}_network", tari_workspace)
+}
+
+pub async fn try_create_container(
+    image: ImageType,
+    fully_qualified_image_name: String,
+    tari_workspace: &str,
+    config: LaunchpadConfig,
```

Comment:
Pretty sure you can get away with a reference here

Reply:
Changes are applied.